Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -1,43 +1,47 @@
+    Enable policy to require PIM elevation from SAW for admin roles in Azure subscriptions
 
 # Prerequisites:
     'Owner' or 'User Access Administrator' role is required at Subscription level.
 
 # Steps performed by the script:
     To remediate:
         1. Validate and install the modules required to run the script.
-        2. Get the critical roles 'Owner', 'Contributor', 'User Access Administrator' where the Conditional Access policy is not configured in a Subscription.
+        2. Get the roles where the Conditional Access policy is not configured in a Subscription.
         3. Back up details of currently configured Conditional Access policy that are to be remediated.
-        4. Configure Conditional Access (CA) policy for 'Owner', 'Contributor', 'User Access Administrator' roles in the Subscription.
+        4. Configure Conditional Access (CA) policy for eligible roles in the Subscription.
 
     To rollback:
         1. Validate and install the modules required to run the script.
         2. Get the roles where the Conditional Access policy was confiured and, are to be rolled back.
-        3. Disable Conditional Access (CA) policy for 'Owner', 'Contributor', 'User Access Administrator' roles in the Subscription.
+        3. Disable Conditional Access (CA) policy for eligible roles in the Subscription.
 
 # Instructions to execute the script:
     To remediate:
         1. Download the script.
         2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
-        3. Execute the script to configure conditional access policy for 'Owner', 'Contributor', 'User Access Administrator' roles in the Subscription. Refer `Examples`, below.
+        3. Execute the script to configure conditional access policy for eligible roles in the Subscription. Refer `Examples`, below.
 
 # Examples:
     To remediate:
         1. To review all the roles where Conditional Access policy is not configured:
                 Configure-ConditionalAccessPolicyForPIM -SubscriptionId '00000000-xxxx-0000-xxxx-000000000000' -DryRun [-PerformPreReqCheck]
 
-        2. To configure Conditional Access (CA) policy in Subscription, from a previously taken snapshot::
+        2. To configure Conditional Access (CA) policy in Subscription, from a previously taken snapshot:
                 Configure-ConditionalAccessPolicyForPIM -SubscriptionId '00000000-xxxx-0000-xxxx-000000000000' -FilePath 'C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\2022010101121\CA_Policy_Details\CA_Policy_Details.csv'
                 
-        3. To configure Conditional Access (CA) policy for All (Owner, Contributor, User Access Administrator) roles in Subscription:
+        3. To configure Conditional Access (CA) policy for all the critical roles (all built-in except a few and all custom roles) in Subscription:
```

Comment:
all built-in except a few and all custom roles --> all built-in and custom roles except few built-in roles

Reply:
Fixed as suggested.